style.css

```
@@ -1,10 +1,121 @@
+	cursor: pointer;
+	color: #64748b;
+	transition: color 0.2s;
+}
+
+.copy-btn:hover{
+	color: #667eea;
+}
+
+@media(max-width:768px) {
+	.palette-container{
+		grid-template-columns: repeat(auto-fit,midmax(100px, 1fr));
```

Biome (1.9.4)

[error] Unexpected unknown function: midmax

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)